<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MonitorClient</class>
 <widget class="QWidget" name="MonitorClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MonitorClient</string>
  </property>
  <widget class="QLabel" name="label_image">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>63</y>
     <width>421</width>
     <height>371</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_warning">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>warning:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_test">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>120</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_user">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>361</width>
     <height>271</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QSplitter" name="splitter_name">
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_name">
     <property name="text">
      <string>UserName: </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_name"/>
   </widget>
   <widget class="QSplitter" name="splitter_pw">
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_pw">
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_pw"/>
   </widget>
   <widget class="QSplitter" name="splitter_log">
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QPushButton" name="pushButton_login">
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_regist">
     <property name="text">
      <string>Regist</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_ConnectServer">
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_return">
    <property name="text">
     <string>Return</string>
    </property>
   </widget>
  </widget>
  <widget class="QSplitter" name="splitter_screen">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>440</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QPushButton" name="pushButton_shot">
    <property name="text">
     <string>ScreenShot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_stop">
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
